Include/Acidanthera/Library/OcBootManagementLib.h

```diff
@@ -19,6 +20,12 @@
 #include <Protocol/LoadedImage.h>
 #include <Protocol/AppleBeepGen.h>
 #include <Protocol/OcAudio.h>
+#include <Protocol/GraphicsOutput.h>
+#include <Protocol/AppleUserInterface.h>
+
+#define __UEFI_MULTIPHASE_H__
+#define __PI_MULTIPHASE_H__
+#include <Pi/PiDxeCis.h>
```

Comment:
Remove these three, replace `Uefi.h` with `PiDxe.h` (makes me wonder whether this should live in a different file)

Reply:
Seems I can used Uefi.h _and_ PiDxe.h in the same file, if I need to, with no hacks - so have done that. Thank you.

Reply:
(Let me know if any apparent problems with that, though as it compiles and works, I guess not.)